sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/ReviewSqlQueriesForSecurityVulnerabilitiesTest.cs

```diff
@@ -103,13 +103,13 @@ string Property2
             Directory.GetFiles(ucfgPath).Select(GetProtobufMethodId).Should().BeEquivalentTo(
                 new[]
                 {
-                    "foo.dll;Program.get_Property1()",
-                    "foo.dll;Program.get_Property2()",
-                    "foo.dll;Program.set_Property2(mscorlib;string)",
-                    "foo.dll;Program..ctor(mscorlib;string)",
-                    "foo.dll;Program.op_Addition(foo.dll;Program,mscorlib;string)",
-                    "foo.dll;Program.Method1(mscorlib;string)",
-                    "foo.dll;Program.Method2()",
+                    "Program.Property1.get",
+                    "Program.Property2.get",
+                    "Program.Property2.set",
```

Comment:
The `value` argument is not in the method signature, but that's fine because in C# you cannot declare two properties with the same name (the return type is not included in the method resolution rules)